### PR TITLE
Auth 4.3.0 docs

### PR DESCRIPTION
**Pull request by pieterlexis**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: docs/changelog/4.3.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Changelogs for 4.3.x
+====================
+
+.. changelog::
+  :version: 4.3.0-alpha1
+  :released: 10th of December 2019
+
+  Compared to the last release candidate, one more bug has been fixed.
+
+  The LMDB backend is incomplete in this version. Slaving zones works, loading zones with pdnsutil works, but more fine grained edits (using edit-zone, or the REST API) fail. We hope to fix this soon in a 4.2.x release.
```
Comment (pieterlexis) [inline docs/changelog/4.3.rst]: pretty sure this isn't the case anymore?

Comment (Habbie) [reply docs/changelog/4.3.rst]: lazy copy, will fix

**Comment by Habbie** [comment]
rebased

**Comment by Habbie** [comment]
> Looks good. I'd suggest removing the `auth: ` prefix in several entries of the ChangeLog, but that's not a big deal.

fixed, thanks!

**Pull request merged**